po/it.po

```diff
@@ -7934,7 +7935,7 @@ msgstr ""
 
 #: ../gramps/gen/lib/ldsord.py:105
 msgid "Canceled"
-msgstr ""
+msgstr "Annullato"
```

Comment:
Do you have a specific reference that says that the LDS church uses this translation? If not, please leave this translation out from this change.